scripts/e2e.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+#
+# Copyright contributors to the Hyperledger Fabric Operator project
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at:
+#
+# 	  http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# step 1. create namespace
+kubectl create namespace u4a-system;
```

Comment:
we can make `e2e.sh` as a `installer.sh`  which can:
- kind/unkind a two/three nodes' kind cluster with arguments
- install/upgrade/uninstall components with arguments
    -  `--all` : by default
    -  `--u4a`
    - `--baas`
    - `--addon dashboard,kubelogin`

Reply:
Good idea, next version to implement.